### PR TITLE
fix: Explore hover dropdown doing weird things

### LINKED ISSUE #6427: Explore hover dropdown doing weird things
https://github.com/kodadot/nft-gallery/assets/90852205/fd42c6b4-928d-44b3-b10f-59121ad426d6

### PR DESCRIPTION
**Pull request by Jarsen136**
## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6427

### CODE REVIEW

File: components/navbar/ExploreDropdown.vue [hunk 1]
```diff
@@ -1,24 +1,26 @@
 <template>
-  <NeoDropdown :triggers="!isMobileDevice ? ['hover'] : []">
+  <NeoDropdown animation="" :triggers="!isMobileDevice ? ['hover'] : []">
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/navbar/ExploreDropdown.vue:2]: I don't think issue is from animation but maybe from hover box content

**Comment by Jarsen136** [comment]
> > There is a margin gap between the button and the dropdown menu, which would make the menu hard to hover. So I remove the gap in this PR.

> Nice but you remove the gap for all dropdown...

I have tried to apply the change to that dropdown but failed. The style would always apply to all dropdown.

File: components/navbar/ExploreDropdown.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jarsen136) [reply components/navbar/ExploreDropdown.vue:2]: Well it did from the animation.

**Comment by roiLeo** [comment]
> I have tried to apply the change to that dropdown but failed. The style would always apply to all dropdown. `$dropdown-menu-margin`

for example this would work:
```css
:deep .navbar-explore .o-drop__menu {
  margin: 0;
}
```

Capture d’écran 2023-07-19 à 8 32 59 AM

Maybe try to add `height: 100%` to `.o-drop` and then flex, align-items: center

**Comment by Jarsen136** [comment]
> > I have tried to apply the change to that dropdown but failed. The style would always apply to all dropdown. `$dropdown-menu-margin`
> 
> for example this would work:
> 
> ```css
> :deep .navbar-explore .o-drop__menu {
>   margin: 0;
> }
> ```
> 
> Capture d’écran 2023-07-19 à 8 32 59 AM
> Maybe try to add `height: 100%` to `.o-drop` and then flex, align-items: center

Thanks. It works well to remove the margin gap from the dropdown. I have update the code.

**Review by roiLeo — APPROVED** [review]
I don't like to see fixed width/height but it's working as excepted

✅ code lgtm

**Pull request merged**